README.md

```diff
@@ -1,17 +1,32 @@
-# SpatialOS Unreal GDK Starter Project
+# SpatialOS GDK for Unreal Starter Project
 
-## Introduction
+* **LICENSE:** Use of the contents of this repository is subject to the [Improbable license](LICENSE.md)
+* **REPOSITORY:** github.com/spatialos/UnrealGDKStarterProject
 
-You can use the Starter Project as the basis for developing a game using the SpatialOS GDK for Unreal Engine. To use the Starter Project, you need to follow the [setup guide](https://github.com/improbable/UnrealGDK/blob/master/docs/setup-and-installing.md) for the GDK. We've enabled use of the `ThirdPersonExampleMap` and a controllable character `AStarterGameCharacter` on a SpatialOS deployment.
+This Starter Project is both a reference project and a starting point for building games on SpatialOS and Unreal Engine 4.  The project uses the `ThirdPersonExampleMap` and a controllable character `AStarterGameCharacter` on a SpatialOS deployment.
 
-> Note that the SpatialOS GDK for Unreal Engine is currently in **alpha**, with stability and performance improvements pending.
+For setup instructions, follow the [Get started](https://docs.improbable.io/unreal/latest/get-started/introduction) guide in the GDK for Unreal documentation.
 
-For more information, see the [GDK repository](https://github.com/improbable/UnrealGDK) and its [documentation](https://github.com/improbable/UnrealGDK/blob/master/docs/readme.md).
+For more information, see the [SpatialOS GDK for Unreal documentation](https://docs.improbable.io/unreal/latest/).
```

Comment:
This link and the one just above are the same page at the moment. Is this intended?